py_gtfs_rt_ingestion/tests/test_ingest.py

```diff
@@ -0,0 +1,19 @@
+from ingest import parseArgs, run
+
+def test_argparse():
+    """
+    Test that the custom argparse is working as expected
+    """
+    dummy_file_path = 'my/fake/path'
+
+    dummy_arguments = ['--fname', dummy_file_path]
+    args = parseArgs(dummy_arguments)
+
+    assert args.fname == dummy_file_path
+
+def test_file_conversion():
```

Comment:
this is the stub where we're going to do the bulk of our testing. obviously, nothing happening right now, but i hope to build this out in earnest on monday.